daq/host.py

```diff
@@ -47,16 +48,31 @@ class MODE:
     LONG = 'long'
     MERR = 'merr'
 
+def _ip_change_verify(host):
+    if len(host.all_ips) <= 1:
+        return None
+    return len(set(map(lambda ip: ip["ip"], host.all_ips))) > 1
+
+_DEFERRABLE_CHECKS = [
+    {
+        "name": "ipchange",
+        "action": lambda host: host.gateway.execute_script('new_ip', host.target_mac),
```

Comment:
can you create a helper function for this so it's like host.gateway.set_new_ip(host.target_mac) or similar?  I'd rather not expose the mechanism like this.